### PR TITLE
Update size of input on Google Cal block

### LINKED ISSUE #17326: Calendar Block: increase input field width
> As the input can take a URL or iFrame, I wonder if we should make it a wider width. The input helps to afford the data needed a little better that way.

image

----

p8oabR-y3-p2#comment-4615

### PR DESCRIPTION
**Pull request by scottsweb**
Updates the Google Calender block UI to be more consistent with other blocks. 

Before:

94680500-1de12d80-0322-11eb-98c2-bbcfca322f75

After:

Screenshot 2020-10-16 at 11 02 53

with comparison to Calendly

Fixes #17326 

#### Changes proposed in this Pull Request:

* Style only

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

N/A

#### Testing instructions:

* Pull this PR
* Create a new Google Calendar block and check the UI 
* Compare it with the Calendly UI 

#### Proposed changelog entry for your changes:

* N/A

### CODE REVIEW

File: extensions/blocks/google-calendar/editor.scss [hunk 1]
```diff
@@ -16,7 +16,14 @@
 	}
 
 	&-embed-form-editor {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		flex-wrap: wrap;
```
Comment (anomiex) [inline extensions/blocks/google-calendar/editor.scss:22]: Is this bit necessary? The element you're targeting seems to already get these styles.

When I add only the two styles below (lines 25–26) to the textarea, it seems to work as intended.

**Review by anomiex — CHANGES_REQUESTED** [review]

File: extensions/blocks/google-calendar/editor.scss (hunk 1 above)
Comment (scottsweb) [reply extensions/blocks/google-calendar/editor.scss]: Good catch, I can see these being set elsewhere too:

Screenshot 2020-10-19 at 09 59 49

I wasn't expecting the wrapper to already be `flex`.

**Review by anomiex — APPROVED** [review]
LGTM.

**Comment by jeherve** [comment]
r215653-wpcom

**Pull request merged**